_posts/en/newsletters/2019-05-21-newsletter.md

```diff
@@ -0,0 +1,221 @@
+      side discussion about the security of relaying Bitcoin data---in
+      short, the Bitcoin protocol is already designed to trustlessly
+      accept data from random peers, so non-net relay doesn't
+      necessarily change the trust model.
+
+   - A [panel][mcf future ln] by Will O'Beirne, Lisa Neigut, Alex Bosworth with
+     moderation by Leigh Cuen discussing the future of LN, mostly the
+     short-term and medium-term conclusion of current development
+     efforts surrounding the LN 1.1 specification.  There are no hyped
+     claims in this discussion, but a simple description of how LN can be
+     expected to improve in ways that make it easier for users and
+     businesses to adopt.
+
+## Bech32 sending support
+
+*Week 10 of 24 in a [series][bech32 easy] about allowing the the people
+you pay to access all of segwit's benefits.*
+
+{% comment %}<!-- weekly reminder for harding: check Bech32 Adoption
+wiki page for changes -->{% endcomment %}
+
+Up until this point in our series encouraging wallets and services to
+support sending to bech32 native segwit addresses, we've focused
+almost exclusively on technical information.  Today, this section
+expresses an opinion: the longer you delay implementing bech32 sending
+support, the worse some of your users and potential users will think of
+your software or service.
+
+> "They can only pay legacy addresses."<br>
+> "Oh.  Let's look for another service that supports current technology."
+
+Services that only support legacy addresses are likely to become a cue
+to users that minimal development effort is being put into maintaining
+their Bitcoin integration.  We expect that it'll send the same signal to
+users as a website in 2019 that's covered in Shockwave/Adobe Flash
+elements and that claims it's best viewed in Internet Explorer 7 (or
+see an even more [imaginative comparison][nullc bank analogy] written by Gregory
+Maxwell.)
+
+Bech32 sending is not some experimental new technology that still needs
+testing---native segwit unspent outputs currently hold [over 200,000
+bitcoins][].  Bech32 sending is also something that's easy to implement
+(see Newsletters [#38][news38 bech32] and [#40][news40 bech32]).  Most
+importantly, as more and more wallets and services upgrade to bech32
+*receiving* by default, it's going to become obvious which other
+services haven fallen behind by not providing sending support.
+
+If you haven't implemented bech32 sending support yet, we suggest you
+try to get it implemented by 24 August 2019 (the two-year anniversary of
+segwit activation).  Not long after that, Bitcoin Core's next release is
+expected to begin defaulting to bech32 receiving addresses in its GUI
+and perhaps also its API methods (see Newsletters [#40][Newsletter #40]
+and [#42][Newsletter #42]).  We expect other wallets to do the
+same---except for the ones that have already made bech32 their default
+(or even their only supported address format).
```

Comment:
Wasabi, for sure.  Also, I think if you enable segwit receiving in Electrum, you can only receive to bech32 addresses (but I haven't tested this myself).

Reply:
Oh, I realized your comment might be asking me to name the wallet(s) here.  I don't want to do that because one of the topics on the 24-week outline is specifically talking about bech32-only wallets and how they're probably going to drag everyone into supporting bech32 sending eventually ("tyranny of the minority").

Reply:
I actually was just curious as I wasn't aware of any wallet that ONLY accepts bech32. We can close this out.